cli/src/update-post-reboot.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
@SomwanshiAnjali 
The general instructions for us are to have lesser batch scripts and more Python code.
Is it possible to convert this to Python code?
Also, move this code under dedicated upgrade directory:
`cli/src/upgrade/update-post-reboot.sh`